templates/impala/base.html

```diff
@@ -56,12 +56,11 @@
         data-collect-timings="{{ url('amo.timing.record') }}:{{ collect_timings_percent }}"
         {% block bodyattrs %}{% endblock %}>
 
-    <div id="tabzilla-wrapper">
+    <div id="main-wrapper">
       <div id="page" class="c">
```

Comment:
Yeah, I guess it's much more reasonable now that we've deleted the script. But there's still loads of other common code. Basically all of the site header stuff, for instance.